tests/unit/model-test.js

```diff
@@ -896,7 +882,7 @@ module('unit/model', function(hooks) {
 
   // TODO: '.setUnknownProperty can update belongs-to relationships'
 
-  test('DS.Models can have relationships into m3 models', function(assert) {
+  skip('DS.Models can have relationships into m3 models', function(assert) {
```

Comment:
This is skipped, because a DS.Model will have further expectation from the model they refer to (I think they try to set an inverse relationship on the model). I've skipped for not to decide whether we want to impose the requirements from DS.Model onto all model datas in order to support interoperability or this should be handled separately from model data.